research/object_detection/dataset_tools/create_ava_tf_record_for_context.py

```diff
@@ -0,0 +1,400 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Code to download and parse the AVA Actions dataset for TensorFlow models.
```

Comment:
This description looks like a copy paste from the file above. Please change this to describe generating tf examples for Context R-CNN. Description of how to use this in conjunction with `generate_embedding_data.py` and `add_context_to_example.py` would be useful